lib/src/widgets/movie_info_detail.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+
+import '../utils/movie_dimensions.dart';
+
+class MovieInfoDetail extends StatelessWidget {
+  const MovieInfoDetail({
+    Key? key,
+    required this.title,
+    required this.releaseDate,
+    required this.isForAdults,
+    required this.overview,
+  }) : super(key: key);
+
+  final String title;
+  final String releaseDate;
+  final bool isForAdults;
+  final String overview;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        SizedBox(
+          height: MovieDimensions.paddingVerticalSmall,
+        ),
+        Text(
+          '$title',
+          style: TextStyle(
+            fontSize: MovieDimensions.fontSizeLarge,
+            color: Colors.grey.shade50,
+            fontWeight: FontWeight.w500,
+          ),
+          textAlign: TextAlign.center,
+        ),
+        SizedBox(
+          height: MovieDimensions.paddingVerticalSmall,
+        ),
+        Text(
+          '$releaseDate   |   '
+          '${isForAdults ? MovieDimensions.detailsIsAdultText : MovieDimensions.detailsIsNotAdultText}',
```

Comment:
I think you can use a vertical divider instead of use this String format

Reply:
Done